DWC.Blazor/wwwroot/data/developers.json

```diff
@@ -691,4 +691,14 @@
 		"twitter": "https://twitter.com/RancellTapia",
 		"github": "https://github.com/RancellTapia"
 	}
```

Comment:
Be aware that a comma is missing here. This will break the json file.

Reply:
Ok thanks , The solution was updated